README.md

```diff
@@ -21,31 +44,42 @@ In addition, it also supports the following props:
 
 ### Marker
 
-A custom Leaflet marker for an individual location. Marker supports all [Leaflet Marker properties](https://leafletjs.com/reference.html#marker).
+A customizable Leaflet marker for an individual location.
+
+Marker supports all [Leaflet Marker properties](https://leafletjs.com/reference.html#marker).
 
 In addition, it also supports the following props:
 
 | Prop    | Type      | Default value |
 | ------- | --------- | ------------- |
+| `icon`  | `Element` | `undefined`   |
 | `popup` | `Element` | `undefined`   |
 
 #### Customizing the marker
 
-Marker will render whatever is passed to its `icon` prop. By default, `grommet-leaflet` will render this as `<Pin />`. However, this can be customized by passing a `Pin` with a specific `kind` defined by the theme (see [Pin](#pin) for details) or a custom element all together.
+Marker will render whatever is passed to its `icon` prop. By default, `grommet-leaflet` will render this as [Pin](#pin). However, this can be customized by passing a `Pin` with a specific `kind` defined by the theme or a custom element.
 
+##### Default marker
 ```
-<Marker icon={<Pin kind="critical" />} />
+<Marker position={{ lat: 40.532, lng: -105.18 }} />
 ```
 
-OR
+##### Theme defined marker
+```
+<Marker position={{ lat: 40.532, lng: -105.18 }} icon={<Pin kind="critical" />} />
+```
 
+
+##### Custom marker
 ```
-<Marker icon={<MyCustomPin />} />
+<Marker position={{ lat: 40.532, lng: -105.18 }} icon={<MyCustomPin />} />
 ```
 
 #### Adding a popup to the marker
 
-A popup appears when a marker is clicked. To add a popup, pass an element as a child to the marker. For ease and consistency, Marker will automatically wrap your element in the required Leaflet Popup component and a themed Popup container. The grommet-leaflet Popup styling is defined in `theme.popup`.
+A popup is a container in which to present addtional information about a location. A popup appears when a marker is clicked.
+
+To add a popup, pass an element as a child to the marker. For ease and consistency, Marker will automatically wrap your element in the required Leaflet Popup component and a themed Popup container. The grommet-leaflet Popup styling is defined in `theme.popup`.
```

Comment:
The popup should be passed as a prop to Marker instead of as a child

Reply:
Thanks. Fixed. Was focussing on general structure and flow, but good to catch and correct in real time. Thank you.